inst/bslib-scss/spacer.scss

```diff
@@ -1,8 +1,9 @@
 $spacer: 1rem !default;
+$bslib-spacer: $spacer !default;
 
 :root {
   // Controls default spacing in layout containers (e.g, layout_columns())
-  --bslib-spacer: #{$spacer};
+  --bslib-spacer: #{$bslib-spacer};
```

Comment:
I'm tempted to rename this `--bslib-gap-spacer` but that's definitely too big of a change for right now. In general, though, I think we should be careful moving forward to use this variable for adjusting spacing between elements. 

I know I used it in some sidebar CSS (that I've adjusted in this PR) for something other than inner-item spacing. I don't think there are remaining places where this happens, but I did want to call it out for future reference.